src/diart/functional.py

```diff
@@ -1,4 +1,6 @@
-from typing import Union, Optional, List, Literal, Iterable, Tuple
+from typing import Union, Optional, List, Iterable, Tuple
+from typing_extensions import Literal 
```

Comment:
There's also a `from typing import Literal` in `sinks.py`, could you please make this change there as well?